### DISCUSSION TITLE
Alguien sabe que debiamos hacer para hoy

### DISCUSSION THREAD

**Discussion by JolisteR** [Launch-X-Latam/MisionFrontEnd#39] [Q&A]
alguien sabe que se dejo como tarea ayer para hoy
saben si hay que leer algo o investigar o solamente era una inttroduccion lo de ayer

**Comment by Pablo-Jaramillo** [comment]
Revisar las carpetas de INTRO y HTML, y si tienes preguntas, apuntarlas en alguna nota para hacerlas al Mission Commander mañana miércoles 28 de Septiembre.

**Comment by doguedogue** [comment]
Checate la ruta del front-end 🚀🚀🚀

https://github.com/Launch-X-Latam/MisionFrontEnd
